### PR TITLE
feat: support wot-design-ui and nutui-uniapp

### PR DESCRIPTION
**Pull request by ModyQyW**
### Description 描述

支持 wot-design-ui 和 nutui-uniapp 作为项目组件库。

## Summary by CodeRabbit

- **New Features**
    - Added new UI component libraries: `wot-design-ui`, `nutui-uniapp`, and `wot-design-uni`.
    - Introduced functionality for handling user input and displaying customized messages.
    - Enhanced CI workflow with new flags for "wot" and "nut" UI options.

- **Dependencies**
    - Updated dependencies to include `nutui-uniapp` and `wot-design-uni`.

- **Documentation**
    - Updated `README.md` to reference the new UI component list.

### CODE REVIEW

**Review by yang1206 — APPROVED** [review]

File: template/UI/nut/vite.config.js.data.mjs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    ...oldData,
+    plugins: oldData.plugins.some(plugin => plugin.id === 'autoImport')
+      ? oldData.plugins.flatMap(plugin =>
+        plugin.id === 'autoImport' ? [{ id: plugin.id, importer: plugin.importer }, autoImportNutUiPlugin] : plugin,
+      )
+      : oldData.plugins.flatMap(plugin =>
+        plugin.id === 'uni' ? [autoImportNutDesignUiPlugin, plugin] : plugin,
+      ),
+  }
+}
```
[the remark this answers is not in the document]
Comment (FliPPeDround) [reply template/UI/nut/vite.config.js.data.mjs:21]: 这个ai好厉害

**Comment by FliPPeDround** [comment]
https://github.com/uni-helper/create-uni/actions/runs/9232741603/job/25404337665?pr=41

nutui-uni的ci没有通过😭

@yang1206 cc

**Review by Moonofweisheng — APPROVED** [review]

**Comment by yang1206** [comment]
> [uni-helper/create-uni/actions/runs/9232741603/job/25404337665?pr=41](https://github.com/uni-helper/create-uni/actions/runs/9232741603/job/25404337665?pr=41)
> 
> nutui-uni的ci没有通过😭
> 
> @yang1206 cc



> [uni-helper/create-uni/actions/runs/9232741603/job/25404337665?pr=41](https://github.com/uni-helper/create-uni/actions/runs/9232741603/job/25404337665?pr=41)
> 
> nutui-uni的ci没有通过😭
> 
> @yang1206 cc

#42

File: template/UI/nut/src/App.vue [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<script setup>
+import { onLaunch } from '@dcloudio/uni-app'
+
+onLaunch(() => { })
+</script>
+
+<style>
+@import 'nutui-uniapp/styles/index';
```
Comment (FliPPeDround) [inline template/UI/nut/src/App.vue]: 这里看起来有问题
我将这里删掉了，组件样式依旧引入了
会有什么其他问题吗？
@yang1206

**Review by FliPPeDround — APPROVED** [review]
目前ci都通过

File: template/UI/nut/src/App.vue (hunk 2 above)
Comment (yang1206) [reply template/UI/nut/src/App.vue]: 这里包含了公共的sass工具函数，还有图标和动画的样式，是必须要导入的

Comment (FliPPeDround) [reply template/UI/nut/src/App.vue]: 谢谢🙏，已经添加上了

**Pull request merged**